src/http/routes.js

```diff
@@ -63,6 +63,14 @@ router.get(
   hasVerifiedAccount,
   controllers.nurseUser.getSingleNurseUser,
 );
+router.put(
+  '/nurse/:nurseId',
+  authMiddleware,
+  hasUserPrivledge([HOSPITAL_ADMIN_USER.toLowerCase(), WARD_USER.toLowerCase()]),
```

Comment:
Only a nurse user should be able to update his/her details

Reply:
fixed